### PR TITLE
Add BC fix for EZP-28145 and Symfony < 3.4

### PR DESCRIPTION
**Pull request by wizhippo**

### CODE REVIEW

File: src/DependencyInjection/Compiler/KernelPass.php [hunk 1]
```diff
@@ -28,13 +28,17 @@ public function process(ContainerBuilder $container)
...
-        $arguments[0] =  array_values(array_filter($arguments[0], function ($argument) {
-            if ($this->isCachePurger($argument)) {
-                return false;
-            }
-            return true;
-        }));
-        $container->getDefinition('cache_clearer')->setArguments($arguments);
+
+        // BC Symfony < 3.4
+        if (is_array($arguments[0])) {
```
Comment (alongosz) [inline src/DependencyInjection/Compiler/KernelPass.php]: Ok, but let's keep early return convention, so
```php
if (!is_array($arguments[0])) {
    return;
}
```

Comment (alongosz) [reply src/DependencyInjection/Compiler/KernelPass.php]: nvm :)

**Comment by alongosz** [comment]
:+1:

File: src/DependencyInjection/Compiler/KernelPass.php (hunk 1 above)
Comment (andrerom) [reply src/DependencyInjection/Compiler/KernelPass.php]: @alongosz could you review e5f73817ca328669735b279a76ada487b6c03409

Comment (alongosz) [reply src/DependencyInjection/Compiler/KernelPass.php]: e5f7381 approved ;)

**Pull request merged**